tsdb/engine/tsm1/tsm1_test.go

```diff
@@ -113,7 +113,7 @@ func TestEngine_WriteAndReadFloats(t *testing.T) {
 	}
 	tx.Rollback()
 
-	if err := e.Close(); err != nil {
+	if err := e.Engine.Close(); err != nil {
```

Comment:
I renamed `Cleanup()` to `Close()` on the test engine wrapper. There were tests that depended on calling the underlying Engine's `Close()` directly.